bikespace_api/bikespace_api/tests/functional/test_submissions.py

```diff
@@ -123,3 +124,74 @@ def test_post_submissions(flask_app, test_client, submission_id=5):
     )
     assert new_submission.comments == dummy_submission["comments"]
     assert (current_datetime - new_submission.submitted_datetime).total_seconds() < 1
+
+
+def test_get_submission_history(flask_app, test_client):
```

Comment:
Is it possible to rollback a change using this library, if so maybe we can write a test to do that ?